src/translations/en.json

```diff
@@ -702,6 +702,8 @@
     "TabularModificationError": "Error while creating tabular modification",
     "TabularModificationGeneratorSkeletonComment" : "#,HYDRO | NUCLEAR | WIND | THERMAL | SOLAR | OTHER,,,,,,true | false",
     "TabularModificationLoadSkeletonComment": "#,AUXILIARY | FICTITIOUS,,",
+    "TabularModificationShuntSkeletonComment": "\"# For each shunt compensator it is possible to modify either the maximum reactive power (and the type) or the maximum susceptance. In case of conflicting input, the maximum susceptance will be ignored.\"",
```

Comment:
By surrounding the `#` with `"`, paparse considers it as part of the cell's text, In order for the comment line to be ignored you should put the `#` in the first column and the rest of the comment in the second one